var/spack/repos/builtin/packages/py-netcdf4/package.py

```diff
@@ -92,3 +95,8 @@ def setup_build_environment(self, env):
         env.set("NETCDF4_DIR", self.spec["netcdf-c"].prefix)
         env.set("NETCDF4_INCDIR", self.spec["netcdf-c"].prefix.include)
         env.set("NETCDF4_LIBDIR", self.spec["netcdf-c"].prefix.lib)
+
+        # ensure cython uses the MPI compiler
```

Comment:
What we need is to ensure `cython` uses Spack compiler wrapper: #50184.

Reply:
Will that be sufficient in the MPI case?

Reply:
yes